src/auth/decorators/current.user.decorator.ts

```diff
@@ -0,0 +1,9 @@
+import { createParamDecorator, ExecutionContext } from '@nestjs/common';
+
+export const CurrentUser = createParamDecorator(
+  (argument: unknown, ctx: ExecutionContext) => {
```

Comment:
You can use void, so other developers can't pass anything to decorator

```suggestion
  (argument: void, ctx: ExecutionContext) => {
```